docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+[versions of pip, any other tools, etc. go here]
```

Comment:
Is there anything we need to add here so users don't have ARGH WRONG VERSION OF THING crises before they've even started?